relayer/relays/beacon/header/syncer/api/api_response_test.go

```diff
@@ -0,0 +1,17 @@
+package api
+
+import (
+	"encoding/json"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+func TestUnmarshalBlockResponse(t *testing.T) {
+	blockResponse := BeaconBlockResponse{}
+	data, _ := os.ReadFile("web/packages/test/testdata/beacon_block_6815804.json")
```

Comment:
Did you try block `6815803` as well? Were you able to reproduce the error?

Reply:
6815803 is fine.

Reply:
Nice catch! 👏🏻